includes/managers/icons.php

```diff
@@ -42,6 +42,39 @@ private static function get_needs_upgrade_option() {
 		return get_option( 'elementor_' . self::NEEDS_UPDATE_OPTION, null );
 	}
 
+	/**
+	 * @param $icon
+	 * @param $attributes
+	 * @param $tag
+	 * @return bool|mixed|string
+	 */
+	public static function try_get_icon_html( $icon, $attributes = [], $tag = 'i' ) {
+		if ( empty( $icon['library'] ) ) {
+			return '';
+		}
+
+		return self::get_icon_html( $icon, $attributes, $tag );
```

Comment:
Always prefer to call current class method by using the `static` instead of `self`